hera_qm/tests/test_ant_metrics.py

```diff
@@ -222,28 +139,6 @@ def test_antpol_metric_sum_ratio():
     assert crossPolRatio == {(0, 'x'): .5, (0, 'y'): .5}
 
 
-def test_average_abs_metrics():
-    metric1 = {(0, 'x'): 1.0, (0, 'y'): 2.0}
-    metric2 = {(0, 'x'): 3.0, (0, 'y'): -4.0}
-    metricAbsAvg = ant_metrics.average_abs_metrics(metric1, metric2)
-    assert np.isclose(2.0, metricAbsAvg[(0, 'x')], atol=1e-10)
-    assert np.isclose(3.0, metricAbsAvg[(0, 'y')], atol=1e-10)
-    metric3 = {(0, 'x'): 1}
-    pytest.raises(KeyError, ant_metrics.average_abs_metrics, metric1, metric3)
-
-
-def test_compute_median_auto_power_dict(lowlevel_data):
```

Comment:
This tests a function that felt so trivial that I rewrote it in the two places it was used as a single dictionary comprehension.